core/src/test/resources/rewrite/RedundantBraces.stat

```diff
@@ -159,3 +160,27 @@ object a {
     } // end
   }
 }
+<<< redundant enclosing braces in str interpolation
+object a {
+  def b(value: String) = s"Hello ${value}"
+}
+>>>
+object a {
+  def b(value: String) = s"Hello $value"
+}
+<<< not redundant enclosing braces in str interpolation
+object a {
+  def b(foo: Int) = s"Hello ${foo}bar"
+}
+>>>
+object a {
+  def b(foo: Int) = s"Hello ${foo}bar"
+}
+<<< enclosing braces in str interpolation
+object a {
+  def b(d: Int) = s"Hello ${d.toString}"
```

Comment:
I don't think the `def b(d: Int) = ` part is necessary. Maybe a block is sufficient
```
{ s"$a" }
```